### DISCUSSION TITLE
Enhance release process and cadence

### DISCUSSION THREAD

**Discussion by Tatsinnit** [Azure/aks-periscope#132] [Ideas]
Some Idea/thoughts share: 💡

I think we should stick to current release practise rather then hookign with every merge to main branch of we can scope if a release is created than only trigger.

Some sample steps we could do:

* The image for this repo is published in `MCR` one way we can do this is to add a creds via GitHub `env var` for publishing reasons.
* But then we should really re-visit the `admin` access for this project.

Any further thoughts are welcome, please feel free to add otherwise I will move this to discussion.

**Comment by Tatsinnit** [comment]
This done as part of #151

**Comment by peterbom** [comment]
Unlocked as this sounds as if it relates to the discussion in #233 (which makes the MCR workflow the one which performs the GitHub release, publishing to GHCR and publishing to MCR).

We need to consider how the workflow is triggered (e.g. from the creation of a Tag, or manually), and how this ties in with updating the changelog (see [guidance](https://keepachangelog.com/en/1.0.0/).
